package.json

```diff
@@ -10,17 +10,26 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.11",
```

Comment:
This package is deprecated and unnecessary. Remove.

Reply:
Removed this unnecessary package